### PR TITLE
feat: Updates docs link & instructions

### PR DESCRIPTION
**Pull request by darigovresearch**
Would resolve issues #43 & #42 if merged.

Also reduces duplication of content in the readme to make the documentation the golden source and avoid synchronisation drift.

### CODE REVIEW

**Comment by drspangle** [comment]
Someone beat you to this PR. I'm not sure if we want to totally wipe out the documentation from the README, maybe we should include a few basic details. Either way, if you can update this PR based on the latest HEAD then I'll go ahead and merge it.

**Comment by darigovresearch** [comment]
@drspangle updates on how to build the docs has been appended to the new updated readme, leave it to you to decide what you wish to take out of the readme.

Not a problem in this case but would just be careful on the order you do pull requests. In this case this PR came before the other one which was merged, odds are you just went from top to bottom in the PR page. Not sure if it may create conflicts in the future or if a guideline for contributing and what the procedure is for merging (for other maintainers that you may delegate to in the future) may be helpful in the future when there is a more complex contribution/the community submissions become more active.

**Review by drspangle — APPROVED** [review]
This seems fine to me. I think you've just got to resolve the merge conflict and you're all set. I'm fine with this being the authoritative version of these files.

**Review by drspangle — APPROVED** [review]

**Pull request merged**
